### PR TITLE
fix: typos in README

### PR DESCRIPTION
**Pull request by 7yon**
Resolves https://github.com/Backbase/.github/issues/7

### CODE REVIEW

**Review by jjjasper — DISMISSED** [review]

File: profile/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Backbase Open Source
 
-Welcome to Backbase's open source organisation. Regardless to if you are a customer, partner or employee you can create and share with everyone in the Backbase Community via this organisation.
+Welcome to Backbase's open source organisation. Regardless of if you are a customer, partner, or employee you can create and share with everyone in the Backbase Community via this organisation.
```
Comment (ugur-bb) [inline profile/README.md:3]: Maybe `Regardless of if ...` -> `Regardless if` or `Regardless of whether`? (Because is sounds little weird otherwise.)

Comment (ugur-bb) [reply profile/README.md:3]: The heading is `Backbase Open Source`, not `Backbase's Open Source`; hence, would be `Backbase Open Source organisation` instead.

**Review by ugur-bb — APPROVED** [review]

File: profile/README.md (hunk 1 above)
Comment (7yon) [reply profile/README.md]: I agree, done ✅

**Review by ugur-bb — DISMISSED** [review]

File: profile/README.md (hunk 1 above)
Comment (craigwalkeruk) [reply profile/README.md]: Would you mind while updating to also change the links?

https://backbase.io/developers/

and

https://backbase.io/learning/

Also, I believe there are 2 versions of this text, one for the public view and one for the members. So this should be copied their too.

Thank you

Comment (7yon) [reply profile/README.md]: @craigwalkeruk links are updated ✅

[I searched for the title of the README on GitHub to find a second page/text](https://github.com/search?q=Welcome+to+Backbase%27s+open+source+organisation.+Regardless+to+if+you+are+a+customer%2C&type=code) but this is the only file I found so far. 
Maybe you could share a link to the 2nd file that should be updated as well?

Comment (7yon) [reply profile/README.md]: Hi @ugur-bb @craigwalkeruk, 
I adressed all comments, except one about the second version of this text that needs to be updated. Unfortunately I couldn't find it myself. Could you please help with it?

File: profile/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Backbase Open Source
 
-Welcome to Backbase's open source organisation. Regardless to if you are a customer, partner or employee you can create and share with everyone in the Backbase Community via this organisation.
+Welcome to Backbase Open Source organisation. Regardless if you are a customer, partner, or employee you can create and share with everyone in the Backbase Community via this organisation.
 
-You'll find repo's here directly maintained by or Backbase teams and essestial tools use as part of our developer experaince. See [learn.backbase.com](https://learn.backbase.com) to pick up a course on our best practices and [developer.backbase.com](https://developer.backbase.com/) to see the offical developer documentaion.
+You'll find repo's here directly maintained by our Backbase teams and essential tools used as part of our developer experience. See [backbase.io/learning](https://backbase.io/learning/) to pick up a course on our best practices and [backbase.io/developers](https://backbase.io/developers/) to see the official developer documentation.
```
Comment (santam85) [inline profile/README.md:5]:
```suggestion
Some of the repositories you will find here are directly maintained by our Backbase teams, as they contain essential tools used as part of our developer experience. See [backbase.io/learning](https://backbase.io/learning/) to pick up a course on our best practices and [backbase.io/developers](https://backbase.io/developers/) to see the official developer documentation.
```

File: profile/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 
-Also you'll also find commuity created connections to 3rd parties and accelerators shared by projects that can be used to speed up projects, these are not supported by Backbase's offical SLA but by the community. For more info see our [github organisation contribution guidelines](https://github.com/Backbase/.github/blob/main/CONTRIBUTING.md)
+You'll also find community created connections to 3rd parties and accelerators shared by projects that can be used to speed up projects, these are not supported by Backbase's official SLA but by the community. For more info see our [github organisation contribution guidelines](https://github.com/Backbase/.github/blob/main/CONTRIBUTING.md)
```
Comment (santam85) [inline profile/README.md:7]:
```suggestion
You'll also find community created 3rd party connectors and accelerators that can be used to speed up projects. All the code found in this space is not supported according to the official Backbase SLA, but by the community. For more info see our [github organisation contribution guidelines](https://github.com/Backbase/.github/blob/main/CONTRIBUTING.md)
```

**Pull request merged**